### ISSUE TITLE
add default read column order from s3 file in S3.to_rds()

### ISSUE THREAD

**Issue by malinaa96** [kfk/grizly#377]
In grizly 0.2 when we were loading data from s3 to rds we were using QFrame so we knew what was the column order and by default the copy statement was using these column names. We need similar thing in `S3.to_rds` so that we don't get errors when the order of columns in S3 is different than in Redshift. 

We already have `column_order` parameter in which we can pass the right order of the columns but we don't want to do
```python
column_order = qf.get_fields(aliased=True)
```
in every workflow. The order should be pulled directly from S3 file.

**Comment by malinaa96** [comment]
This one needs to be done in `0.3.4` release as it already caused a lot of issues when the column order in S3 file wasn't the same as in RDS table.

We ca use this 
```python
from boto3 import resource

obj = resource("s3").Object('acoe-s3', 'test.csv')
header = obj.get()['Body'].read().splitlines()[0].decode('utf-8')
col_names = header.split('\t')
```
to get column order. I will add it next week.

**Comment by malinaa96** [comment]
Fixed in 0.3.4
